src/config/readConfigFile.ts

```diff
@@ -5,13 +5,36 @@
 import fs from "fs";
 import { buildReadConfigFileCallback } from "./buildReadConfigFileCallback";
 import { OpenResult } from "./openConfig";
+import path from "path";
+import { CheckedConfig } from "./execute";
 
 // eslint-disable-next-line functional/no-return-void, @typescript-eslint/prefer-readonly-parameter-types
 export type Accept = (value: OpenResult | PromiseLike<OpenResult>) => void;
 
 export const configFile = ".folder-structure-lint.json";
-export const message = `Could not open the config file "${configFile}"`;
+export const jsConfigFile = ".folder-structure-lint.js";
+export const message = `Could not open the config file "${configFile}" or "${jsConfigFile}"`;
 
+type MaybeCheckedConfig = Partial<CheckedConfig>;
+
+// eslint-disable-next-line max-lines-per-function
 export function readConfigFile(accept: Accept) {
-  fs.readFile(configFile, "utf8", buildReadConfigFileCallback(accept));
+  if (fs.existsSync(jsConfigFile)) {
```

Comment:
I plan to break this out into a different file and follow your convention with the coding style however I just wanted to get your approval on this way of doing it first

Reply:
@dmastrorillo, I didn’t see anything that looked concerning. I like it.